msgcntr/messageforums-app/src/webapp/jsp/pvtMsgReplyAll.jsp

```diff
@@ -267,9 +267,7 @@
 				</div>
 		  </div>
 
-
-
-			<h4><h:outputText value="#{msgs.pvt_message}" /></h4>
+			<h4><h:outputText value="#{msgs.pvt_star}" styleClass="reqStar"/><h:outputText value="#{msgs.pvt_message}" /></h4>
```

Comment:
Aligning is off on the file, I decided to set it as the associated inputRichText